docker/worker/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#set -e
+
+if [ -z $1 ]; then
+  echo "Error: define the number of workers"
+  echo "Usage: $0 <number-of-workers>"
+  exit
```

Comment:
Maybe a non-zero return code would make sense here.

Reply:
I think 2 is common here

Reply:
Before going too far with custom code I recommend to use "getopt", see https://github.com/os-autoinst/openQA/blob/master/script/configure-web-proxy#L3 as an example